draft/implementation-notes/index.html

```diff
@@ -489,6 +489,11 @@ <h2>Version Numbering</h2>
           following <code>v</code> is a zero then the number format is zero padded to fixed length, otherwise it
           is simply an integer.
         </p>
+        <p>
```

Comment:
Perhaps:

"In the OCFL, zero padding is not recommended to avoid having to make arbitrary choices of padding length or to place limits on the number of versions sorted. However, on most operating systems this has the effect of showing directories sorted in lexical order, (where "10" comes before "2") rather than natural, numerical order. Numeric sorting may instead be accomplished with the following commands:
 - `ls | sort -n` (most systems)
 - `ls -v` (GNU Linux)
 - `dir | sort` (Windows Command prompt)"
---

Reply:
That seems like too much detail to me, we don't go to that level anywhere else. Also, I don't think those commands work for filenames with with the preceding `v`

Reply:
The implementation notes are (I thought) where details like this would live. "It doesn't do lexical sorting by default *as written in the spec* but here's how you can achieve this."

The commands work for me (slight adjustment to `sort` args):

```
mac$ ls
v1   v10  v100 v2   v20  v3
mac$ ls | sort -n -k1.2
v1
v2
v3
v10
v20
v100
---
linux$ $ ls
v1  v10  v100  v2  v20  v3
linux$ ls -v
v1  v2  v3  v10  v20  v100
---
(Volunteers to test in Windows?) 
```

Reply:
Cool that adding `-k1.2` fixes it for sort, I didn't know about that option.

I think we need to have a editors' discussion about the level of detail though. I'm not opposed the idea of providing such detail but we don't yet do it anywhere and it would have implications for the maintenance of the implementation notes document.

Reply:
My first reaction was similar to yours, @zimeon : the suggestion seemed heavy on low-level details.

If we consider the implementation-notes as a scratch-pad for related ideas, the detailed approach is probably ok. My inclination is to keep the implementation-notes a bit more general... but I don't think the alternative is a critical issue.

Reply:
I think we should keep it simple, right now the detail feels like here's how you do unix ls

Reply:
OK, I can withdraw my review, reluctantly, but with reservations.

I think we all agree that for OCFL implementations, the devil will be in the implementation details. It's all well and good to define a spec for the content layout, but the *risk* will always be with content processing. 

The idea for the implementation notes, as I remember it, was as a place to put our thinking about interactions with "the object in motion."

Without specific, concrete interaction examples, people are going to produce valid OCFL Objects, but how they get there will be uncontrolled. I'm not saying we need to be prescriptive about how people have to implement an OCFL client library, but I am saying that we, at least, should give people examples of how to do something so that they can either copy-and-paste, or have an understanding of how they might be deviating from the intended design.

If we're couching the implementation notes in the same abstract language as the spec, is there any other place where we can give people real, concrete examples of how to interact with an OCFL Object?